Makefile

```diff
@@ -1,39 +1,13 @@
-imports = \
-	@testable import ApplicativeRouterHttpPipelineSupportTests; \
-	@testable import ApplicativeRouterTests; \
-	@testable import CssTests; \
-	@testable import CssResetTests; \
-	@testable import HtmlCssSupportTests; \
-	@testable import HtmlPlainTextPrintTests; \
-	@testable import HttpPipelineTests; \
-	@testable import HttpPipelineHtmlSupportTests; \
-	@testable import UrlFormEncodingTests;
-
 xcodeproj:
-	xcrun --toolchain swift swift package generate-xcodeproj --xcconfig-overrides Development.xcconfig
-	xed .
-
-bootstrap: common-crypto-mm xcodeproj xcodeproj-mm
-
-common-crypto-mm:
-	-@sudo mkdir -p "$(COMMON_CRYPTO_PATH)"
-	-@echo "$$COMMON_CRYPTO_MODULE_MAP" | sudo tee "$(COMMON_CRYPTO_MODULE_MAP_PATH)" > /dev/null
+	xcrun --toolchain swift swift package generate-xcodeproj
```

Comment:
think we could kill this too?

Reply:
Same as other...needed for running iOS and macOS tests from the command line...